apps/src/codebridge/FileBrowser/styles/filebrowser.module.scss

```diff
@@ -14,7 +14,8 @@
 
 .fileBrowserContents {
   overflow-y: auto;
-  scrollbar-color: var(--background-neutral-senary) var(--background-neutral-secondary);
```

Comment:
sorry about the linter changes- it was just a few updates so I decided to keep them